exercises/leap/example.go

```diff
@@ -1,7 +1,10 @@
 package leap
 
-const testVersion = 3
-
-func IsLeapYear(i int) bool {
-	return i%4 == 0 && i%100 != 0 || i%400 == 0
+// IsLeapYear confirms whether a given year is a leap year.
+func IsLeapYear(year int) bool {
+	return year%4 == 0 && year%100 != 0 || year%400 == 0
 }
+
+// This test versioning is specific to Exercism,
+// you don't need to worry about it now.
```

Comment:
Until we remove the test versioning (see https://github.com/exercism/go/pull/808#issue-251732155 `the testVersion–we're going to be able to get rid of this book-keeping in the near future`) I figured in these bonus exercises before it has been explained in `hamming`, we could get away with basically telling people to ignore it all.